### DISCUSSION TITLE
What to choose when installing?

### DISCUSSION THREAD

**Discussion by betekson** [ElPumpo/TinyNvidiaUpdateChecker#240] [Q&A] [answered]
Hello, I have a question, what to select from the list of choices when installing the driver?
![new-update-to-tinynvidiaupdatechecker-fully-customizable-v0-ag6crouf7e6d1](https://github.com/user-attachments/assets/c752430b-7a44-4de0-b068-513b29957a4e)

**Comment by unalignedcoder** [comment]
Same question here. Probably just the Graphics, Audio and PhysX drivers... But the list is a bit intimidating considering many components don't even have a description...

**Comment by Technetium1** [comment]
Run the normal installer and choose custom, see what is already there.

**Comment by HawaiiBeach** [comment, marked as answer]
Guys you already answered your question. You don't know what to choose, which probably means you don't need any of the things.

I would just choose the Graphics Driver, and Optimus, if you're on laptop.
